sktime/utils/results_writing.py

```diff
@@ -80,6 +85,126 @@ def write_results_to_uea_format(path, strategy_name, dataset_name, y_true,
     file.close()
 
 
+def write_dataframe_to_tsfile(data, path, problem_name,
```

Comment:
Regarding the restructuring of utils, this function should probably go into `data_container` and not `results_writing`, no?

Reply:
I'd also set a default argument for `problem_name` so that users don't have to specify it whenever they just want to write out some data

Reply:
hi markus, I asked him to put it here for now, we are delaying all refactoring until you give us the green light 👍